app/src/test/kotlin/no/nav/arbeidsgiver/notifikasjon/kafka_backup/BackupHendelserTests.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.arbeidsgiver.notifikasjon.kafka_backup
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+import no.nav.arbeidsgiver.notifikasjon.KafkaBackup
+import no.nav.arbeidsgiver.notifikasjon.util.EksempelHendelse
+import no.nav.arbeidsgiver.notifikasjon.util.embeddedKafka
+import no.nav.arbeidsgiver.notifikasjon.util.testDatabase
+import java.util.concurrent.atomic.AtomicBoolean
+
+class BackupHendelserTests: DescribeSpec({
+    val database = testDatabase(KafkaBackup.databaseConfig)
+    val backupRepository = BackupRepository(database)
+    val kafka = embeddedKafka()
+    val producer = kafka.newProducer()
+    val consumer = kafka.newRawConsumer()
+    val stop = AtomicBoolean(false)
+    var eventsSent = 0
+    var eventsRead = 0
+
+    describe("write to and read from database") {
+        EksempelHendelse.Alle.forEach {
+            producer.send(it)
+            eventsSent += 1
+        }
+
+        xit("kan lese alle med raw consumer") {
```

Comment:
Denne testen er kommentert ut. Når den kjører nå, så blir den stuck (ser ikke ut til at den finner noen meldinger). 

Er ganske sikker på at den fungerte tidligere. Og den unit-testen med embedded kafka som tester den vanlige consumeren fungerer fortsatt.

Får se på endringer. Men tips tas i mot med takk.

Reply:
funker for meg når jeg kun kjører den og sammen med alle andre 🤔